### PR TITLE
Project `DateTime` get properties

### PR DESCRIPTION
**Pull request by halildurmus**

### CODE REVIEW

File: tool/inputs/interfaces.dart [hunk 1]
```diff
@@ -112,13 +112,13 @@ final windowsRuntimeTypesToGenerate = [
   // changes to the generation script.
   // 'Windows.Foundation.IAsyncAction',
   // 'Windows.Foundation.IAsyncOperation`1',
+  // 'Windows.Globalization.ICalendar',
```
Comment (halildurmus) [inline tool/inputs/interfaces.dart:115]: `ICalendar` should not be autogenerated until we update the generation script to expose `IVectorView` types as `List`.

File: tool/inputs/interfaces.dart [hunk 2]
```diff
@@ -112,13 +112,13 @@ final windowsRuntimeTypesToGenerate = [
...
   // 'Windows.Networking.Connectivity.INetworkInformationStatics',
-  // 'Windows.Networking.IHostName',
   // 'Windows.Storage.IUserDataPathsStatics',
   // 'Windows.Storage.Pickers.IFileOpenPicker',
   //
   'Windows.Foundation.IAsyncInfo',
   'Windows.Foundation.IClosable',
   'Windows.Foundation.IStringable',
-  'Windows.Globalization.ICalendar',
+  'Windows.Networking.IHostName',
```
Comment (halildurmus) [inline tool/inputs/interfaces.dart:123]: `IHostName` can be generated automatically.

**Comment by timsneath** [comment]
Awesome! At some point, we'll need to add support for parameters and setters. The former could be a little more complex, since of course a method could include a DateTime, a HSTRING, and a Vector...

**Comment by halildurmus** [comment]
> Awesome!

Thanks.

> At some point, we'll need to add support for parameters and setters. The former could be a little more complex, since of course a method could include a DateTime, a HSTRING, and a Vector...

Yeah, adding support for parameters is gonna be really tricky. I'm currently working on projecting `IMap` and `IMapView` types. Once I'm done with that I might take a stab at it.

**Pull request merged**